security/docker/Dockerfile.node-agent-k8s

```diff
@@ -1,4 +1,5 @@
 FROM istionightly/base_debug
 ADD node_agent_k8s /
+RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificates
```

Comment:
I think base image handle this now (added by https://github.com/istio/istio/pull/10949), is there ```/etc/ssl/certs``` in your nodeagent container without adding this line ?

Reply:
#10949 adds `ca-certificates` to `ubuntu:xenial`, whereas we are using `istionightly/base_debug` here.
Does it make sense to change the image to `ubuntu:xenial`?

Reply:
It is ok to use istionightly/base_debug since istionightly/base_debug has more debug tools.

Reply:
consider
```
apt-get --no-install-recommends
```
to avoid installing additional packages that you did not explicitly want.
(c.f. suggestion below)

Reply:
Consider adding:
```
\
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/*
```
cleaning up the apt cache and removing /var/lib/apt/lists helps keep the image size down. 
c.f. https://docs.docker.com/engine/articles/dockerfile_best-practices/

(c.f. suggestion below)

Reply:
```suggestion
RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates && update-ca-certificates && apt-get clean  && rm -rf /var/lib/apt/lists/*
```

Reply:
Done, thanks!